ios/FluentUI.Tests/FontTests.swift

```diff
@@ -63,16 +29,16 @@ class FontTests: XCTestCase {
         let font = UIFont.fluent(fontInfo)
         let otherFontDescriptor = UIFontDescriptor(name: name.appending("-Semibold"), size: size)
         let otherFont = UIFont(descriptor: otherFontDescriptor, size: size)
-        XCTAssertEqual(font, otherFont)
+        XCTAssertEqual(font.fontDescriptor, otherFont.fontDescriptor)
```

Comment:
Why did you change this check?

Reply:
Unfortunately the new way we construct fonts fails Apple's `isEqual` check, despite the fact that every observable external attribute of the font (including its visual appearance!) matches. Since their implementation of equality is opaque to us, the only reasonable workaround I could find was to check the font descriptors, which are simple enumerations of the properties of the font (e.g. weight, size, typeface name).